pkgs/development/compilers/temurin-bin/jdk-darwin-base.nix

```diff
@@ -67,6 +70,7 @@ let
     passthru = {
       jre = result;
       home = result;
+      bundle = "${result}/Library/Java/JavaVirtualMachines/${name-prefix}-${lib.versions.major finalAttrs.version}.jdk";
```

Comment:
For correctness:
`result` should be swapped out for `finalAttrs.finalPackage`.
(Also, the old instances of `result` should probably get replaced as well.)

If you don't want to do this right now, it's fine. (It won't cause a rebuild if we do it in another PR).

Reply:
I included it in this PR because I was also the one to introduce the usage of `finalAttrs` in the first place, but I did separate it out into its own commit. I'm happy to squash it into the previous commit or discard it entirely if you'd prefer